datamap/mv/core/src/test/scala/org/apache/carbondata/mv/rewrite/MVExceptionTestCase.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.carbondata.mv.rewrite
+
+import org.apache.spark.sql.catalyst.analysis.NoSuchTableException
+import org.apache.spark.sql.test.util.QueryTest
+import org.scalatest.BeforeAndAfterAll
+
+class MVExceptionTestCase  extends QueryTest with BeforeAndAfterAll {
+  override def beforeAll: Unit = {
+    drop()
+    sql("create table main_table (name string,age int,height int) stored by 'carbondata'")
+  }
+
+
+
+  test("test mv no base table") {
+    sql("drop datamap if exists main_table_mv")
+    sql("drop table if exists main_table_error")
+
+    try {
+      sql("create datamap main_table_mv on table main_table_error using 'mv' as select sum(age),name from main_table group by name")
+    }
+    catch {
+      case e: NoSuchTableException => assert(true)
+      case e => assert(false)
+    }
```

Comment:
val ex = intercept[NoSuchTableException]{
sql("create datamap main_table_mv on table main_table_error using 'mv' as select sum(age),name from main_table group by name")
}
and check message of ex

Reply:
@QiangCai  done, please review again